fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSchemaScan.java

```diff
@@ -26,31 +29,48 @@
 import org.apache.doris.nereids.trees.plans.RelationId;
 import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
 import org.apache.doris.nereids.util.Utils;
+import org.apache.doris.qe.ConnectContext;
 import org.apache.doris.statistics.Statistics;
 
 import com.google.common.collect.ImmutableList;
 
+import java.util.HashSet;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.Set;
 
 /**
  * PhysicalSchemaScan.
  */
 public class PhysicalSchemaScan extends PhysicalCatalogRelation {
+    // if the schema table only root user readable, we need insert into this set
+    public static final Set<String> BACKEND_TABLE_ONLY_ROOT_READABLE = new HashSet<>();
```

Comment:
I think we should use the auth framework to limit the access, not this tricky method